frontend/src/interfaces/route.ts

```diff
@@ -16,6 +16,14 @@ export interface IRoute {
     isStandalone?: boolean;
 }
 
+export interface ICondensedRoute {
```

Comment:
What is the meaning of "condensed" in this context?

Reply:
Essentially it's just a navigation route. It doesn't have all the properties of IRoute, and it does not have a component because it's not being used to render a route, just populate a navigation menu. It could be clearer: `INavigationMenuItem` ?